runtime/zeitgeist/src/integration_tests/xcm/tests/transfers.rs

```diff
@@ -356,47 +387,51 @@ fn transfer_eth_sibling_to_zeitgeist() {
     Zeitgeist::execute_with(|| {
         let expected = transfer_amount - eth_fee();
         let expected_adjusted = adjusted_balance(eth(1), expected);
+        let expected_treasury = treasury_initial_balance + adjusted_balance(eth(1), eth_fee());
 
         // Verify that remote Alice now has initial balance + amount transferred - fee
         assert_eq!(
-            AssetManager::free_balance(ETH_ID.into(), &ALICE),
+            AssetManager::free_balance(ETH_ID.into(), &alice()),
             zeitgeist_alice_initial_balance + expected_adjusted,
         );
-
         // Verify that fees (of foreign currency) have been put into treasury
         assert_eq!(
             AssetManager::free_balance(ETH_ID.into(), &ZeitgeistTreasuryAccount::get()),
             // Align decimal fractional places
-            treasury_initial_balance + adjusted_balance(eth(1), eth_fee())
+            expected_treasury
         )
     });
 }
 
 #[test]
 fn transfer_eth_zeitgeist_to_sibling() {
```

Comment:
Maybe this should test that truncation works as expected? Completely random comment, but would it make sense to open an issue about this?

Reply:
The decimal adjustments are implicitly tested by verifying adjusted balances on accounts using `adjusted_balance`